<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>440</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>680</width>
    <height>440</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>680</width>
    <height>440</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{background-image: url(:/img/snoopy.jpeg);}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>430</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>430</height>
    </size>
   </property>
   <widget class="QWidget" name="chatwidget" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>440</width>
      <height>410</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>440</width>
      <height>410</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>440</width>
      <height>410</height>
     </size>
    </property>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>420</width>
       <height>250</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>420</width>
       <height>250</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>420</width>
       <height>250</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(114, 159, 207);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fileBtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>90</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>25</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(114, 159, 207);</string>
     </property>
     <property name="text">
      <string>File</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>420</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>420</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>420</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(114, 159, 207);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sendBtn">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>380</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>25</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(114, 159, 207);</string>
     </property>
     <property name="text">
      <string>send</string>
     </property>
    </widget>
    <widget class="QWidget" name="headwidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-10</y>
       <width>440</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>440</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>440</width>
       <height>50</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="headLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </widget>
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>200</width>
      <height>410</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>410</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>410</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(114, 159, 207);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabSpacing">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Freinds">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>30</height>
      </size>
     </property>
     <attribute name="label">
      <string>Friends</string>
     </attribute>
     <layout class="QVBoxLayout" name="friendsLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="addfriendBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>add friend</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="Groups">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>200</width>
       <height>60</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>60</height>
      </size>
     </property>
     <attribute name="label">
      <string>Groups</string>
     </attribute>
     <layout class="QVBoxLayout" name="groupsLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="creategroupBtn">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>create group</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addgroupBtn">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>add group</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
